### ISSUE TITLE
WinDivert 2.0 support

### ISSUE THREAD

**Issue by Terricide** [TechnikEmpire/WinDivertSharp#10]
Any plans to support the new version of WinDivert?

**Comment by TechnikEmpire** [comment]
No. You gain nothing from batching. Same problem exists, packet reordering. The solution is to use 1 thread or implement the reordering minimisation technique described in the performance bug ticket on windivert.

1 thread is basically faster than everything else. The kernel mode driver can only ever execute one thread at a time anyway so faking threading is nothing but a performance killer.

**Comment by Terricide** [comment]
Huh? I don't understand your answer. I was just saying that there is a new version of WinDivert that this library doesn't work with and wondering you were planning on updating the library to support the newer dll.

**Comment by TechnikEmpire** [comment]
2.0 was a refactor to add batching and some other layers. Primarily batching. Batching is useless.

Short answer: no more support for anything I'm writing my own driver.

**Comment by Terricide** [comment]
Ahhhhh.... okay thanks. Makes sense now. 

Thanks
